scripts/datasets/pretrain_corpus/prepare_bookcorpus.py

```diff
@@ -0,0 +1,46 @@
+"""Prepare the bookcorpus dataset that contains raw text file."""
+
+import os
+import time
+import tarfile
+import argparse
+from gluonnlp.utils.misc import download
+
+_DOWNLOAD_URL\
+    = "https://the-eye.eu/public/AI/pile_preliminary_components/books1.tar.gz"

```

Comment:
How about put this url into `scripts/datasets/url_checksums` like https://github.com/dmlc/gluon-nlp/blob/master/scripts/datasets/url_checksums/wikipedia.txt

Reply:
I have already added it to bookcorpus.txt with hash sha1. I guess now it is more clear.